web/admin/src/components/ResultListTable.svelte

```diff
@@ -177,7 +179,9 @@
   >Live</wa-switch
 >
 
-{#if tableData && tableData.length > 0}
+{#if loading}
+  <Loader />
+{:else if tableData.length > 0}
```

Comment:
```suggestion
{:else}
```

Does not hurt to always show. Less confusing.